### ISSUE TITLE
If I dont't start up the plugin when status is "Playing", it's can not get right Score(Give a random value)I had updated to ORTDP 1.4.5

### ISSUE THREAD

**Issue by Someone999** [OsuSync/OsuRTDataProvider-Release#4]

**Comment by KedamaOvO** [comment]
I can't reproduce it on my computer, but I guess there may be a problem with the second score address. 
Now I removed the second score address. 
Can you help me test it for me?
https://1drv.ms/u/s!AiDHMZIqThV1hv0VK2GvP69sDFkXFg

**Comment by Someone999** [comment]
Could you change a website ? It's seems like happened error when I open that.

**Comment by Someone999** [comment]
Could you change a website ? My browser can not open this. Or take it on GitHub?

**Comment by KedamaOvO** [comment]
https://pan.baidu.com/s/1uxEXknYbeGVwxHDfaZvuWQ  1r1v

**Comment by Someone999** [comment]
![screenshot1714](https://user-images.githubusercontent.com/45456778/56751662-335f3900-67b9-11e9-9165-5dbd3c4cacb7.jpg)

**Comment by Someone999** [comment]
![批注 2019-04-26 002429](https://user-images.githubusercontent.com/45456778/56751918-b41e3500-67b9-11e9-89d2-79042f508752.jpg)

**Comment by Someone999** [comment]
It seems not be solved
![screenshot1715](https://user-images.githubusercontent.com/45456778/56752241-67872980-67ba-11e9-8eb2-ac1eac8e41ed.jpg)

**Comment by Someone999** [comment]
Sometimes it will works after restart the program

**Comment by Someone999** [comment]
![批注 2019-04-26 004942](https://user-images.githubusercontent.com/45456778/56753487-38be8280-67bd-11e9-909a-2ea229ddb93d.jpg)
This two Address?

**Comment by Someone999** [comment]
Sometimes the value of score is one of the address with out initialization.

**Comment by Someone999** [comment]
It does not work until I open Cheat Engine

**Comment by Someone999** [comment]
I restarted game , and try to debug it.
When debugging that , I found this
![批注 2019-04-26 011541](https://user-images.githubusercontent.com/45456778/56754953-df585280-67c0-11e9-9c70-a5a09640d3b1.jpg)
This address is different from the address Cheat Engine found
![批注 2019-04-26 011725](https://user-images.githubusercontent.com/45456778/56755018-1595d200-67c1-11e9-83cc-050035035677.jpg)

**Comment by Someone999** [comment]
After restart the program it work again. This issue may because address of osu! will change when every boot?

**Comment by Someone999** [comment]
But sometimes this also doesn't work after restart the program.

**Comment by KedamaOvO** [comment]
> After restart the program it work again. This issue may because address of osu! will change when restart?

Yes. Because of the JIT, the base address changes every time it starts.

**Comment by KedamaOvO** [comment]
This problem should not happen because Hit Count also relies on this base address.

**Comment by KedamaOvO** [comment]
Is your Osu version up to date?

**Comment by KedamaOvO** [comment]
Are you sure you are using 1.4.6-tset now?

**Comment by KedamaOvO** [comment]
1.4.6-test removes the second base address and should be no different from CE.
![image](https://user-images.githubusercontent.com/9410177/56756743-087ae200-67c5-11e9-9f8c-dddb688069b9.png)

**Comment by Someone999** [comment]
Oh,I forget to tell you , The Hit Objcets are getted from RTPPD by mmf.

**Comment by Someone999** [comment]
I use this plugin by reference , not by sync,I confired that version is 1.4.6

**Comment by Someone999** [comment]
I use this by reference.And confirmed that the version is 1.4.6

**Comment by Someone999** [comment]
I forgot to tell you , I get hit objects by RTPPD not by ORTDP

**Comment by Someone999** [comment]
I will test that after home.

**Comment by Someone999** [comment]
Sometimes Hit Objects is work but score not work.

**Comment by Someone999** [comment]
Now seems good.

**Comment by Someone999** [comment]
I think I need to restart my program every time osu! restart. But if not , only the value of score is incorrect
![screenshot1717](https://user-images.githubusercontent.com/45456778/56798116-c1cecb80-6848-11e9-95a0-9346dfb66dea.jpg)

**Comment by Someone999** [comment]
osu  is also up-to-date

**Comment by KedamaOvO** [comment]
Please check the version of ORTDP in Sync. 
![image](https://user-images.githubusercontent.com/9410177/56798935-6c93b980-684a-11e9-99ff-cead786ca1d3.png)

**Comment by Someone999** [comment]
![批注 2019-04-26 180204](https://user-images.githubusercontent.com/45456778/56800401-7e2a9080-684d-11e9-9293-21d05bbb4435.jpg)

**Comment by KedamaOvO** [comment]
I can't understand what's going on in ORTPD right now. 😥

**Comment by Someone999** [comment]
Not a big problem
I only need to restart it.

**Comment by Someone999** [comment]
Thank you.

**Comment by Someone999** [comment]
Init PlayFinder Failed?
